pkg/scaffold/helm/main.go

```diff
@@ -36,16 +36,24 @@ func (m *Main) GetInput() (input.Input, error) {
 const mainTmpl = `package main
 
 import (
+	"os"
+
 	hoflags "github.com/operator-framework/operator-sdk/pkg/helm/flags"
 	"github.com/operator-framework/operator-sdk/pkg/helm"
 
 	"github.com/spf13/pflag"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 )
 
 func main() {
+	logf.SetLogger(logf.ZapLogger(false))
```

Comment:
This is getting set here and also within `helm.Run` (same goes for ansible). We should probably only call it once.

1. We can leave it this way and remove it from `helm.Run`, but we'd then have to do this everywhere we call `helm.Run`
2. We can remove this line and rely on `helm.Run` calling it, which should still work for the log in line 55. That's a bit awkward because line 55 would have a hidden dependency on something happening in `helm.Run`
3. Maybe there are other options for passing the logger to `helm.Run` or exporting the logger used in `helm.Run` as a package variable so we could use it here (e.g. `helm.Log.Error(err, "")`) to make the dependency more explicit.

I think I lean toward option 1, but there might be a better solution along the lines of option 3 that I'm not thinking of. Thoughts?

Reply:
I’m in agreement with option 1, as I’d rather have the user able to make direct logging changes.